wazo_call_logd/database/dao.py

```diff
@@ -58,11 +58,14 @@ def new_session(self):
         finally:
             self._Session.remove()
 
-    def get_by_id(self, cdr_id):
+    def get_by_id(self, cdr_id, tenant_uuids):
         with self.new_session() as session:
-            cdr = session.query(CallLogSchema).options(joinedload('participants'),
-                                                       subqueryload('source_participant'),
-                                                       subqueryload('destination_participant')).get(cdr_id)
+            query = session.query(CallLogSchema).options(joinedload('participants'),
+                                                         subqueryload('source_participant'),
+                                                         subqueryload('destination_participant'))
+            query = self._apply_filters(query, {'tenant_uuids': tenant_uuids})
+            query = query.filter(CallLogSchema.id == cdr_id)
+            cdr = query.one_or_none()
```

Comment:
By curiosity, since `cdr_ir` is the primary_key, why not continue to use `.get(cdr_id)` instead of `filter` + `one_or_none`

Reply:
get() will also returns call logs that not from my allowed tenants.